lib/bootloader_setup.pm

```diff
@@ -994,6 +994,14 @@ sub zkvm_add_disk {
             enter_cmd("# copying image...");
             $svirt->add_disk({file => $hdd_path, backingfile => 1, dev_id => 'a'});    # Copy disk to local storage
         }
+        # in MicroOS or SLEM we need to add ignition drive to proceed with image basic setup
```

Comment:
That's not really MicroOS specific at all, it's just support for multiple disks...

I suggest to do it like `configure_blockdevs` in `os-autoinst/OpenQA/Qemu/Proc.pm`.

Reply:
Nice point, let me check this code. Thanks